packages/rocket-file-uploads-aws/src/file-uploaded.ts

```diff
@@ -0,0 +1,49 @@
+import { RocketFilesUserConfiguration } from 'rocket-file-uploads-types/dist'
+
+interface AWSEventS3 {
+  s3SchemaVersion: string
+  configurationId: string
+  bucket: {
+    name: string
+    ownerIdentity: any
+    arn: string
+  }
+  object: {
+    key: string
+    size: number
+    eTag: string
+    sequencer: string
+  }
+}
+
+interface AWSEvent {
+  eventVersion: string
+  eventSource: string
+  awsRegion: string
+  eventTime: string
+  eventName: string
+  userIdentity: [Record<string, unknown>]
+  requestParameters: [Record<string, unknown>]
+  responseElements: [Record<string, unknown>]
+  s3: AWSEventS3
+}
+
+export function getMetadataFromRequest(request: unknown): AWSEvent {
+  const events = (request as any)['Records'] as Array<AWSEvent>
+  return events[0]
+}
```

Comment:
This can crash, also typing could be improved like:
```suggestion
export function getMetadataFromRequest(request: { Records: Array<AWSEvent> }): AWSEvent {
  const metadata = events.Records?.[0]
  if (!metadata) {
    throw new Error('Metadata arrived empty in the request records')
  }
  return metadata
}
```